components/scrollable-container/script.js

```diff
@@ -183,5 +185,29 @@ class ScrollableContainer extends HTMLElement {
         });
     }
 }
+
+/**
+ * Will add styles for the non Coherent browsers that are disabling the native scrollbar
+ */
+function addDisabledNativeScrollbarStyles() {
+    const style = document.createElement('style');
+    style.innerHTML = `
+        .guic-native-scroll-disabled {
+            -ms-overflow-style: none;
+            scrollbar-width: none;
+        }
+        .guic-native-scroll-disabled::-webkit-scrollbar {
+            display: none;
+        }`;
+    document.head.appendChild(style);
+}
+
+if (!components.isBrowserGameface() && !components.nativeScrollDisabledStylesAdded) {
+    addDisabledNativeScrollbarStyles();
+    // We are doing this to prevent readdition of the styles needed for the native scrollbar to be not visible
+    // This problem is produced when multiple components are bundled the scrollable container.
```

Comment:
```suggestion
    // We are doing this to prevent readdition of the styles needed for the native scrollbar to not be visible.
    // This problem is produced when multiple components have the scrollable container bundled.
```
or
...are bundling the scrollable container.

Reply:
Fixed